control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -2510,18 +2506,20 @@ func TestRun_ValidateLoginToken_PrimaryDatacenter(t *testing.T) {
 			})
 			require.NoError(t, err)
 
-			tok, _, err := client.ACL().Login(&api.ACLLoginParams{
-				AuthMethod:  authMethodName,
-				BearerToken: jwtToken,
-				Meta:        map[string]string{},
-			}, &api.WriteOptions{})
-			require.NoError(t, err)
+			if len(c.Roles) == 1 {
```

Comment:
Can we create an assertion to ensure the roles created for terminating gateways are what we have supplied in the above? We are skipping that assertion right now right?

Reply:
agg.  that was a remnant.  thanks for catching this!